store/tikv/oracle/oracles/pd.go

```diff
@@ -112,11 +150,24 @@ func (o *pdOracle) getTimestamp(ctx context.Context) (uint64, error) {
 	return oracle.ComposeTS(physical, logical), nil
 }
 
-func (o *pdOracle) setLastTS(ts uint64) {
-	lastTS := atomic.LoadUint64(&o.lastTS)
+func (o *pdOracle) setLastTS(ts uint64, txnScope string) {
+	_, exist := o.lastTSMap[txnScope]
```

Comment:
There should be only one key for "global", but `txnScope` can be either "global" or "".